core-web/libs/dotcms-webcomponents/src/components/dot-html-to-image/dot-html-to-image.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, Prop, h, Host, Event, EventEmitter, State } from '@stencil/core';
+import '@material/mwc-circular-progress';
+
+@Component({
+    tag: 'dot-html-to-image',
+    styleUrl: 'dot-html-to-image.scss',
+    shadow: false
+})
+export class DotHtmlToImage {
+    @Prop({ reflect: false })
+    value = '';
+
+    @Prop({ reflect: false, mutable: true })
+    height = '';
+
+    @Prop({ reflect: false, mutable: true })
+    width = '';
+
+    @Event() pageThumbnail: EventEmitter<File>;
+    @State() previewImg: string;
+
+    iframeId = `iframe_${Math.floor(Date.now() / 1000).toString()}`;
+    loadScript = `
+        html2canvas(document.body, {
+            height: IMG_HEIGHT, // The height of the canvas
+            windowHeight: IMG_HEIGHT, // Window height to use when rendering Element
+            width: IMG_WIDTH, // The width of the canvas
+            windowWidth: IMG_WIDTH, // Window width to use when rendering Element
+        }).then((canvas) => {
+            canvas.toBlob((blob) => {
+                const fileObj = new File([blob], 'image.png');
+                window.parent.postMessage({
+                    iframeId: '${this.iframeId}',
+                    previewImg: canvas.toDataURL(),
+                    fileObj
+                }, '*')
+            });
+        }).catch(function (error) {
+            console.error('oops, something went wrong!', error);
```

Comment:
improve this message ?

Reply:
Shouldn't we show a `popup`? So the user can see the `message`. We can also use an `alert`.